proto/panacea/datadeal/v2alpha2/tx.proto

```diff
@@ -43,7 +43,7 @@ message MsgCreateDealResponse {
 // MsgSellData defines the Msg/SellData request type.
 message MsgSellData {
   uint64 deal_id = 1;
-  string verifiable_cid = 2;
+  bytes verifiable_cid = 2;
```

Comment:
If I rollback to `[]byte`, is it okay to use base64 encoding in keeper function?

Reply:
In addition, the `DataSale`'s `verifiableCID` type is `[]byte`. Is it need to change a `DataSale`'s `verifiableCID` type too?

Reply:
The `verifiable_cid` we will use is the `CID` of IPFS.
This `CID` is used in the form of a string.
https://docs.filebase.com/ipfs/ipfs-cids
So it seems better to use strings instead of byte arrays.

Reply:
Ok, I understand it. Thanks!